spec/services/laa_reference_updater_spec.rb

```diff
@@ -33,4 +36,28 @@
       record
     end
   end
+
+  context 'with no maat reference' do
+    let(:maat_reference) { nil }
+
+    before do
+      allow_any_instance_of(LaaReferenceUpdater).to receive(:dummy_maat_reference).and_return('A1000000')
```

Comment:
```suggestion
      ActiveRecord::Base.connection.execute('ALTER SEQUENCE dummy_maat_reference_seq RESTART;')
```
Would it be easier if we reset the sequence, and then we can drop the private method spec.

Reply:
Yes, this is much better